/**
* DO NOT EDIT THIS FILE.
* See the following change record for more information,
* https://www.drupal.org/node/2815083
* @preserve
**/
(function (_ref) {
  var throwError = _ref.throwError,
    behaviors = _ref.behaviors;
  behaviors.testErrors = {
    attach: function attach() {
      throwError(new Error('A manually thrown error.'));
    }
  };
})(Drupal);